data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -173,23 +133,21 @@ public Object process(org.apache.hadoop.hive.ql.lib.Node nd, Stack<org.apache.ha
 		ASTNode pt = (ASTNode) nd;
 		switch (pt.getToken().getType()) {
 			case HiveParser.TOK_QUERY:
-//				System.out.println("\"got query\" = " + pt);
-				System.out.println("\nQuery " + ++subquerySeq + " parsed");
-//				printDot(pt);
-
+				LOGGER.info("\nQuery " + ++subquerySeq + " parsed");
 				getTableLineage(pt);
 
 				for (String inputTable : inputTableList) {
-					System.out.println("inputTable = " + inputTable);
+					LOGGER.info("inputTable = " + inputTable);
 				}
 
+				//adding to outputTableList and outputNodes
 				if (outputTableList.size() == 0) {
 					String outputTable = "TEMP_" + UUID.randomUUID();
 					outputTableList.add(outputTable);
 					if (outTableNodes.size() == 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1155)